server/server.js

```diff
@@ -59,7 +59,7 @@ var bedPath;
  * Reads the configuration file and gets the file paths.
  */
 function config() {
-  var tokens = fs.readFileSync('server/config')
+  var tokens = fs.readFileSync('config')
```

Comment:
Please remove this change. The server is currently started as server/server.js in the genomes root folder. Or you can change the readme to reflect your change here.

Jiaming changed this to server/server.js originally. Im not sure which way is better.